aws/GitOps/clusters/optional-test-infra-tekton/namespaces/kubeflow-test-infra/eks-cluster-create.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: eks-cluster-create
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "aws, eks"
+    tekton.dev/displayName: "EKS Cluster Create"
+spec:
+  description: |
+    Create an EKS cluster.
+
+    This Task can be used to create an EKS cluster in an AWS account and fetch a kubeconfig that
+    can be used (in a context with kubectl) to make requests to the cluster.
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin.
+  - name: version
+    default: "1.17"
```

Comment:
Maybe we can use 1.18 directly?

Reply:
I've just finished the kfserving testing in 1.18 and it's ok to use it as default.